packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/VersionTrackingTests.kt

```diff
@@ -81,7 +81,6 @@ class VersionTrackingTests {
         realm.activeVersions().run {
             assertEquals(1, all.size)
             assertEquals(1, allTracked.size)
-            assertNull(notifier)
```

Comment:
The check is a left over from when doing https://github.com/realm/realm-kotlin/issues/1234 where I tried to avoid initializing the notifier until actually listening to objects. Doing this caused the Realm to not receive initial sync updates, so was removed again. 

We are now initializing the notifier in `RealmImpl`, but it is not happening synchronuously `Realm.open` and until now it always was null here. Don't know what has changed under the hood, but I have removed the assertion as we cannot guarantee anything. `main` has an updated assert that it is either `null` or asserted.